ui/src/app/machines/views/MachineDetails/MachineSummary/OverviewCard/StatusCard/StatusCard.test.tsx

```diff
@@ -116,8 +117,87 @@ describe("StatusCard", () => {
       </Provider>
     );
 
-    expect(wrapper.find("[data-testid='error-description']").text()).toBe(
+    expect(screen.getByTestId("error-description")).toHaveTextContent(
       "machine is on fire"
     );
   });
+
+  it("does not display a sync status for deployed machines with hardware sync disabled", () => {
+    const machine = machineDetailsFactory({
+      enable_hw_sync: false,
+      status: NodeStatus.DEPLOYED,
+    });
+    const store = mockStore(state);
+
+    render(
+      <Provider store={store}>
+        <MemoryRouter
+          initialEntries={[{ pathname: "/machines", key: "testKey" }]}
+        >
+          <StatusCard machine={machine} />
+        </MemoryRouter>
+      </Provider>
+    );
+
+    expect(
+      screen.queryByText(/Periodic hardware sync/)
```

Comment:
Same suggestion as the PR about using a `Label` enum. If the copy changes this test will still pass.

Reply:
This is a good point, I'll look into this.

Reply:
I'll look into all these label-related comments separately, as part of a single task: https://github.com/canonical-web-and-design/app-tribe/issues/797